packages/create-turbo/src/utils/isDefaultExample.ts

```diff
@@ -0,0 +1,5 @@
+export const DEFAULT_EXAMPLES = new Set(["basic", "default"]);
+
+export function isDefaultExample(example: string): boolean {
+  return DEFAULT_EXAMPLES.has(example);
+}
```

Comment:
Seems very specific to the "internal-transform", so putting it in a general utility seems like an unnecessary indirection

Reply:
Yea good point, I'll clean this up a bit more